cmd/root.go

```diff
@@ -74,8 +80,16 @@ var RootCmd = &cobra.Command{
 	Long:          BannerColor.Sprintf("\n%s", consts.Banner),
 	SilenceUsage:  true,
 	SilenceErrors: true,
-	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		setupLoggers(logFmt)
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		logger := logrus.StandardLogger() // don't use the global one to begin with
+		if envLogOutput, ok := os.LookupEnv("K6_LOG_OUTPUT"); ok {
+			logOutput = envLogOutput
+		}
```

Comment:
this seems wrong - if you put this here, the environment variable will have precedence over the CLI flag, right?

Reply:
:facepalm: ... got it wrong ... I thought that was the precedence ... 
The other way around will be harder though >.> ... maybe I will leave it for some other time in that case ...

Reply:
how is it harder? my 1-line code suggestion would have worked, even if it's not pretty 
```go
var logOutput string = os.GetEnv("K6_LOG_OUTPUT")
```

Reply:
Unfortunately ... not as the cli flags parsing overwrites it ...

Reply:
Ah, true, you also had to set its default value to whatever its current `os.GetEnv("K6_LOG_OUTPUT")` value was, which was confusing when you run `k6 run --help`... This messes like: https://github.com/loadimpact/k6/blob/0e94653c9954c17b2b1a480b456e23ec120feff2/cmd/root.go#L126-L130

 I like your solution better anyway :+1: